cryoet_data_portal_neuroglancer/state_generator.py

```diff
@@ -180,8 +180,22 @@ def generate_image_layer(
     rms: float | None = None,
     is_visible: bool = True,
     has_volume_rendering_shader: bool = False,
+    twodee_contrast_limits: tuple[float, float] | None = None,
+    threedee_contrast_limits: tuple[float, float] | None = None,
+    volume_rendering_is_visible: bool = False,
+    volume_rendering_gain: float = -7.8,
```

Comment:
I think having this be a static value would not be ideal for the diverse cases of tomograms. We should look into generating this dynamically. 

@uermel would you have any recommendation on how to address this?